Python runtime metrics (#4370)
DataDog/documentation: content/en/tracing/advanced/runtime_metrics.md

{{% /tab %}}
{{% tab "Python" %}}

> Python metrics collection can be enabled with one environment parameter when running with `ddtrace-run`:

Reviewer: Python metrics -> Runtime metrics

Reviewer 2: ```suggestion
Runtime metrics collection can be enabled with one environment parameter when running with `ddtrace-run`:
```

> Python metrics collection can be enabled with one environment parameter when running with `ddtrace-run`:

Reviewer: <div class="alert alert-info">
This feature is currently in <strong>BETA</strong>.
Reach out to <a href="/help">the Datadog support team</a> to be part of the beta.
</div>

Reviewer 2: ```suggestion
This feature is currently in **BETA**. Reach out to the [the Datadog support team][1] to be part of the beta.
```

* Environment Variable: `DD_RUNTIME_METRICS_ENABLED=true`

> Python metrics can be viewed in correlation with your Python services. See the [Service page][1] in Datadog.

Reviewer: Python metrics -> Runtime metrics

Reviewer 2: ```suggestion
Runtime metrics can be viewed in correlation with your Python services. See the [Service page][1] in Datadog.
```

> **Note**: For the runtime UI, `ddtrace` >= [`0.24.0`][2] is supported.

Reviewer: ddtrace-py we mean?

Reviewer 2: The repo is `dd-trace-py`, but you install it as `ddtrace`

{{% /tab %}}
{{% tab "Python" %}}

> This feature is currently in **BETA**. Reach out to the [the Datadog support team][1] to be part of the beta.

Reviewer 2: ```suggestion
<div class="alert alert-info">
This feature is currently in <strong>BETA</strong>.
Reach out to <a href="/help">the Datadog support team</a> to be part of the beta.
</div>
```

{{% /tab %}}

> {{% tab "Python" %}}

Reviewer 3: beta front matter is missing

<div class="alert alert-info">
This feature is currently in <strong>BETA</strong>.
Reach out to <a href="/help">the Datadog support team</a> to be part of the beta.

> </div>

Reviewer 3: beta front matter is missing

> Runtime metrics collection can be enabled with one environment parameter when running with `ddtrace-run`:

Reviewer 3: other languages call out using dogstatsd and port 8125. Are we not using that here?

{{% /tab %}}
{{% tab "Python" %}}

{{< get-metrics-from-git "python" >}}

Along with displaying these metrics in your APM Service Page, Datadog provides a [default Python Runtime Metrics Dashboard][1] with the `service` and `runtime-id` tags that are applied to these metrics.

> [1]: https://app.datadoghq.com/dash/integration/xyz/python-runtime-metrics

Reviewer 2: TODO: We need to replace `xyz` with the actual dashboard id.

Reviewer: Good catch. Python Dashboard's up at https://app.datadoghq.com/dash/integration/30267/python-runtime-metrics right?

Reviewer 2: ```suggestion
[1]: https://app.datadoghq.com/dash/integration/30267/python-runtime-metrics
```
